roles/cfg_openwrt/templates/common/config/wireless.j2

```diff
@@ -4,8 +4,13 @@
 # Wifi Config is derived from wireless profile: '{{ wireless_profile }}'
 {% for wd in wireless_devices | default([]) %}
   {% set wd_id = 'radio' + loop.index0|string %}
-  {% set wd_config = profile['devices'] | selectattr('radio', 'contains', wd['name']) | first %}
   {% set wd_ifaces = profile['ifaces'] | default([]) | selectattr('radio', 'contains', wd['name']) %}
+  {% if 'devices' in profile %}
+    {% set wd_config = profile['devices'] | default([]) | selectattr('radio', 'contains', wd['name']) | first %}
+  {% else %}
+    # No config provided, use defaults
+    {% set wd_config = {} %}
+  {% endif %}
```

Comment:
I think this chunk is quite ugly, but I'm not finding a jinja2/ansible solution that is nicer. I thought `| default({})` might do it, but I was getting errors.